### DISCUSSION TITLE
Filename too long error, unique to this PC.

### DISCUSSION THREAD

**Discussion by DanteDercksen** [community/community#51089] [Repositories]
### Select Topic Area

Bug

### Body

I keep getting this error when trying to clone a Unity project. I've tried initializing git-lfs to no avail. The specific file has been removed from the branch but when cloning the branch it opens on the main repository and when I try to switch branches it fails with the error: 

error: cannot stat 'Assets/External Art Assets/EffectCore v1.7/packs/StylizedExplosionPack1/prefabs/ShaderGraphVersion/red_originalFire_noSmoke_ShaderGraph/explosion_stylized_medium_originalFireNoSmoke_ShaderGraph.prefab.meta': Filename too long

I've tried cloning the repo to another computer and it works perfectly fine. Please if anyone can help me urgently I would appreciate it.

**Comment by airtower-luna** [comment]
"Filename too long" indicates a limit of your OS, not a file *size* issue. And the filename length limit probably includes the whole path, including any directories the repository is in (which might be different from other computers).
